src/components/BookSearch/common/BorrowButton.js

```diff
@@ -14,15 +16,18 @@ export default function BorrowButton({ rowStyles, book }) {
     userBookManager.isReserved(book.isbn13)
   );
 
+  const [showBorrowModal, setShowBorrowModal] = useState(false);
+
   const handleBorrowClick = (e) => {
     e.stopPropagation();
-    userBookManager.borrowBook(book.isbn13);
-    setIsBookBorrowed(true);
+    // userBookManager.borrowBook(book.isbn13);
+    // setIsBookBorrowed(true);
```

Comment:
zakomentowany kod to największy prześladowca programistycznej ludzkości, unikaj jak ognia, wiem z doświadczenia że potem tego się prawie nie da usunąć, więc póki wiemy  że to nie potrzebne, usuń to tak samo z tym poniżej